sungrow/integrationpluginsungrow.json

```diff
@@ -0,0 +1,272 @@
+                            "name": "currentPower",
+                            "displayName": "Active power",
+                            "type": "double",
+                            "unit": "Watt",
+                            "defaultValue": 0,
+                            "cached": false
+                        },
+                        {
+                            "id": "7bf092a2-da07-11ee-abe9-138e66e6f2a5",
+                            "name": "totalEnergyProduced",
+                            "displayName": "Total energy produced",
+                            "type": "double",
+                            "unit": "KiloWattHour",
+                            "defaultValue": 0.0,
+                            "cached": true
+                        },
+                        {
+                            "id": "80b35fc2-da07-11ee-a73d-5774b082c92b",
+                            "name": "temperature",
+                            "displayName": "Temperature",
+                            "type": "double",
+                            "unit": "DegreeCelsius",
+                            "defaultValue": 0.00,
+                            "cached": false
+                        },
+                        {
+                            "id": "84bcbc58-da07-11ee-b94d-a39792ee6f59",
+                            "name": "frequency",
+                            "displayName": "Frequency",
+                            "type": "double",
+                            "unit": "Hertz",
+                            "defaultValue": 0.00,
+                            "cached": false
+                        }
+                    ]
+                },
+                {
+                    "name": "sungrowMeter",
+                    "displayName": "Sungrow Meter",
+                    "id": "a935e49c-da07-11ee-bd11-3fcf27dc6373",
+                    "createMethods": ["auto"],
+                    "interfaces": [ "energymeter", "connectable"],
+                    "stateTypes": [
+                        {
+                            "id": "d26b59dc-da07-11ee-b494-3781cc3081a7",
+                            "name": "connected",
+                            "displayName": "Connected",
+                            "displayNameEvent": "Connected changed",
+                            "type": "bool",
+                            "defaultValue": false,
+                            "cached": false
+                        },
+                        {
+                            "id": "d777530e-da07-11ee-a526-c7d23dd34f57",
+                            "name": "currentPower",
+                            "displayName": "Current power",
+                            "type": "double",
+                            "unit": "Watt",
+                            "defaultValue": 0.00,
+                            "cached": false
+                        },
+                        {
```

Comment:
If we have the voltage and the current per phase, can we provide also the power per phase? The energy manager can handle both, current and/or power values per phase, but if we would have them, it would make sense to provide them, even by setting the state value to `P = U * I`